src/pybind/mgr/cephadm/module.py

```diff
@@ -2588,6 +2640,18 @@ def add_daemon(self, spec: ServiceSpec) -> List[str]:
             self.events.from_orch_error(e)
             raise
 
+    @handle_orch_error
+    def get_prometheus_access_info(self) -> Dict[str, str]:
```

Comment:
what's the expected use case with returning the root cert here? Is there some place the user might need that?

Reply:
> what's the expected use case with returning the root cert here? Is there some place the user might need that?

This used by the dashboard to get the access info information for prometheus. This is called from `dashboard/controllers/prometheus.py` to perform requests on the prometheus server.